lib/bugsnag/breadcrumbs/breadcrumb.rb

```diff
@@ -0,0 +1,35 @@
+module Bugsnag::Breadcrumbs
+  class Breadcrumb
+    attr_accessor :message
+    attr_accessor :type
+    attr_accessor :meta_data
+    attr_reader :auto
+    attr_reader :timestamp
+
+    def initialize(message, type, meta_data, auto)
```

Comment:
yardoc comments?

Reply:
If we're starting to adopt yardoc in bugsnag-ruby, there should be `@param`, `@return`, etc. comments (see https://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md , but don't follow that first snippet which actually RDoc instead). Also, `#initialize` should probably have an `# @api private` as I don't think end users will need to create a `Breadcrumb`.